nginx/conf.d/api.core.conf.tmpl

```diff
@@ -9,6 +9,15 @@ location ${PATH_PREFIX}/api/ {
                 return 503;
         }
 
+        proxy_pass ${API_SERVICE_URL}${PATH_PREFIX}/api/;
+        include /opt/app-root/etc/nginx.conf.d/proxy.params;
+}
+
+location ${PATH_PREFIX}/api/v1 {
```

Comment:
You may have to add a trailing slash to v1

Reply:
Added.